app/views/restaurants/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Restaurants</h1>
```

Comment:
🔨 @sbwheeler indentation is off, each level of indentation should only be 2 spaces